frontend/src/libs/components/loader/loader.module.scss

```diff
@@ -0,0 +1,83 @@
+// @import "../../../assets/css/variables.scss";
+
+$loader-bg-color: #ffffff80;
+$loader-color: #cceeff;
```

Comment:
Remove the comment. The variables should be imported to the global styles.scss file.
Also, remove the variables from this file.